Task2/src/main/java/LiveLock.java

```diff
@@ -0,0 +1,44 @@
+import java.util.logging.Logger;
+
+public class LiveLock {
+
+    private static final Logger LOGGER = Logger.getLogger(DeadLock.class.getName());
+
+    public static void workWithResourcesIfAvailable(Resource resource1, Resource resource2) {
+        while (true) {
+            try {
+                Thread.sleep(1000);
```

Comment:
try to make this task through synchro block and conditions. Read about wait/notify methods in class Object: 
https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#wait()
https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#notify()